whitepapers/types.md

```diff
@@ -1,9 +1,9 @@
 ---
 published: true
-permalink: /whitepapers/type-transaction-code/
+permalink: /whitepapers/types/
```

Comment:
Do the corresponding links on dataelements.md and the related GitHub issue need to be updated?